sections/sponsor-perks/sponsor-perks.module.scss

```diff
@@ -1,18 +1,37 @@
 @import '../../styles/styles.scss';
 
 .sponsor-perks {
-  background-color: $light_grey;
+  background-color: $secondary;
 
   &-content {
-    @include page-margin;
```

Comment:
i think we still need the page-margin here so the perks section lines up with the rest of the page

<img width="407" alt="Screen Shot 2021-01-18 at 11 13 31 pm" src="https://user-images.githubusercontent.com/19167953/104914161-d1118780-59e2-11eb-8759-18ee4f5d37ee.png">

Reply:
Can we have a call about this?